devlog/2024-11-20-clean-cleanl-clean.md

```diff
@@ -0,0 +1,57 @@
+
+At the start January 2024, Jakub Nugen ([androidnisse](https://github.com/androidnisse)) comes in and start contributing to homepage task too. Worried that we would stepped on each other works, we started a private chat to organize the tasks between ourselves. Until May, the two of us have been chipping away the homepage task which at this point leaving 650 packages without homepage. Then we get more contributors joining. I decided to make a group chat to accomodate all the people joining. It was called "Solus Homepage Team" at the time. More contributors and staff members join the room, then it was renamed as "Solus Cleanup Crew" and moved under official Solus matrix space.
+
+## What is it?
+Solus cleanup crew is a group of bunch of community contributors as well staff members with the goal to bringing up Solus package repository up to current standard. It is a subset of the packaging room on our matrix channel. We organize the work on all the cleanup related tasks.
+
+## Why bother?
+Solus has been updating their tooling and infrastucture since its resurection. While most of the tooling changes can be accomplished by staff members alone, there are things that they cannot do. While many consider Solus repository as small, it still contains more than 6000 packages. Some packages has never even got an update since Solus founded! We want to get rid of all its cruft before we can migrate to a better tooling. That's where the community contributor can come in and help.
+
+For me personally, it is chance to learn something new and fill my time with more purpose. Sometime, I feel like a slob scrolling through things, opening the same three apps over and over or sleeping midday. Technically speaking, doing packaging is still scrolling through things, still opening the same three apps over and over, and I still get my midday nap, but now it is with purpose. I love that what I do can be enjoyed by other people. It is mostly invisible and not even noticed by people, but the chance of it helping people gives me so much joy. Why bother? Because I love it.
+
+## What we accomplished
+We create a few repository wide task that can done with little reading of our packaging guide. The keen-eyed of you may have already seen these tasks. It is always attached at the end of every Sync Updates since July. These are the tasks we did:
```

Comment:
```suggestion
We created a few repository wide tasks that can be done with little reading of our packaging guide. The keen-eyed of you may have already seen these tasks. It is always attached at the end of every Sync Updates since July. These are the tasks we did:
```